### ISSUE TITLE
Rick Dangerous and other "no intro" dumps do not load

### ISSUE THREAD

**Issue by eku** [badda71/uae3DS#6]
I use the latest official version together with the Kickstart ROM 1.3 (md5sum ```82a21c1890cae844b3df741f2762d48d```, 256kB) and try to start [Rick Dangerous](https://archive.org/download/AmigaRomCollectionByGhostware_201711/Rick%20Dangerous.zip). Only the Workbench disk logo is displayed. Nothing happens.

**Comment by badda71** [comment]
You need the overdumped version (512kb). Please check installation instructions: "It must be called kick.rom, be a kickstart 1.3 image, and be the 512KB overdumped version."
Using this rom, Rick Dangerous works fine - at least on my 3DS ...

**Comment by eku** [comment]
Whether 256k or 512k makes no difference to me. The LED for DF0 lights up briefly and then comes the logo of the WB. Is there anything else to do besides "Load disk image"? 

Is it possible to turn on debug messages?

**Comment by badda71** [comment]
Then I guess, your image is broken. I found a working one at https://www.planetemu.net/rom/commodore-amiga-games-adf/rick-dangerous-1989-micro-style-cr-ocl-t-4-ocl-3

**Comment by eku** [comment]
Yeah, that'll start. But this is a cracked version and not a dump of the original disk. ```uae3ds``` does not load many so called ```no intro dump``` disk in ADF format as available from archive.org. Follow the link in my first post.

See also the [interview with Rob Northen](https://codetapper.com/amiga/interviews/rob-northen/) and scroll down to ```Amiga games that use the Rob Northen Copylock Protection System```. I'm pretty sure that the ADF of ```Rick Dangerous``` still contains the copylock as it is a perfect dump of the original disk.

BTW WB 1.3 disk loads with both 256kb and 512kb kick rom as available from [archive.org](https://archive.org/download/verifiedbiosfiles/OGA%20BIOS/Amiga/).

**Comment by badda71** [comment]
Jup, could definitely be the copy protection kicking in ... not sure how I can solve that issue. Guess, you just have to live with playing the cracked games ;-)

**Comment by eku** [comment]
You could report the problem to the project where the emulator source code comes from.

**Comment by badda71** [comment]
That project is long dead. If you can check whether these images are working on Switch UAE, I might have a solution for you (only New3DS).

**Comment by eku** [comment]
If you own a switch with UAE, you could do the test yourself. I linked the ADF in the first post. I don't own one.

**Comment by badda71** [comment]
same here, sorry
